src/components/paragraph/button/index.tsx

```diff
@@ -0,0 +1,23 @@
+import { v4 as uuidv4 } from 'uuid'
+import Link from 'next/link'
+
+const Button = ({ paragraph }): JSX.Element => {
+  if (!paragraph) return
+
+  const {
+    field_button_label: fieldButtonLabel,
+    field_button_link: fieldButtonLink,
+  } = paragraph
+
+  return (
+    <div key={uuidv4()}>
+      <Link href={fieldButtonLink ? fieldButtonLink.uri : ''} passHref>
+        <a className="vads-c-action-link--blue">
+          {fieldButtonLabel ? fieldButtonLabel : ''}
+        </a>
+      </Link>
```

Comment:
I wonder if filtering null values in the parent means we won't need the ternaries anymore? On the other hand if the Button component is being used by a different page the dev doesn't filter null values early on it could cause some broken links so maybe leaving them in is a better idea, idk?

Reply:
@amponce I added more validation and removed the ternaries.